### ISSUE TITLE
Output of Jupyter notebook cells is not intuitively accessible with screen readers

### ISSUE THREAD

**Issue by GauravTheCoder** [microsoft/vscode#175743]
@isidorn 
<b>Bug</b>

Using: latest JAWS and NvdA.
Steps to reproduce:
1. Open a jupyter notebook and make 2-3 code cells.
2. Type Python code in each one and hit control-enter. You hear the audio cue announcing that the code in the cell worked.
3. Tab around(after enabling "Pressing Tab will now move focus to the next focusable element") and move to the cell output. it speaks "read only", but pressing arrow keys reads nothing there.

you have to enable the virtual PC cursor/Browse mode to read the output. I wonder if it's possible to make control+uparrow and control+downarrow work there, similar to the accessible buffer in terminal? Not to mention one single keystroke to jump to the output of the corresponding notebook cell output, that would make data science a lot easier and intuitive for blind people.

Thanks!

VS Code version: Code 1.75.1 (441438abd1ac652551dbe4d408dfcec8a499b8bf, 2023-02-08T21:32:34.589Z)
OS version: Windows_NT x64 10.0.22621
Modes: Restricted
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i7-1255U (12 x 2611)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.68GB (9.58GB free)|
|Process Argv|--crash-reporter-id dd32757f-cd87-4808-9f73-6d6c49ef34b6|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (7)</summary>

Extension|Author (truncated)|Version
---|---|---
python|ms-|2023.2.0
vscode-pylance|ms-|2023.2.40
jupyter|ms-|2023.1.2010391206
jupyter-keymap|ms-|1.1.0
remote-ssh|ms-|0.96.0
remote-ssh-edit|ms-|0.84.0
remote-explorer|ms-|0.2.0

</details>

**Comment by isidorn** [comment]
@GauravTheCoder this is great feedback, thank you very much for opening this issue!

fyi @minsa110 @meganrogge

**Comment by meganrogge** [comment]
@amunger and I are meeting with @jooyoungseo to discuss this issue next Monday

**Comment by amunger** [comment]
I think output accessibility should be improved now with the accessible view, with related instructions in the notebook accessibility help (alt+F1).

**Comment by kds215** [comment]
Hi, can you give some specific details on how to launch Notebook Accessibility Help via Alt+F1  (mac and windows) and possibly the related help (full) instructions? Blind folks would like to know ;-).

**Comment by amunger** [comment]
Open a notebook and press Alt+F1. Is there anything specifically giving you trouble?

**Comment by amunger** [comment]
> There is no info in the Accessibility Help window on how to navigate/access Output cells.

From the accessibility help (that you pasted):

> The Focus Output command (Control+Shift+PageDown) will set focus in the cell's output.

Is that what you're looking for?

**Comment by kds215** [comment]
Yes, Focus Output is the command that a blind person needs to get to the 
‘blue frame’ state of the output cell for the (JAWS) reader to kick in 
and to start reading.

The issue has been that ‘Ctrl+Shift+PgUp/PgDn’ is defined in the 
VScode Keyboard shortcut for Windows doc as ‘Move editor left/right’.
The Focus Output key was not defined on my mac and not on my blind friend’s 
windows laptop. Only after defining the Focus Output keystroke did
my friend feel he made progress. It is just luck that we have stumbled
onto a workable solution for him. Working (blind) in VScode notebooks is
like walking on a path through a jungle of key strokes that are
just NOT reliable/dependable. 

1. reporting more troubles about the Focus Output functionality:
in our join efforts around the “how to navigate to a notebook's output cell?”, 
my blind friend and I have found and reported bug #201862 (Jan24).
However, we have NOT reported the short coming when the output cell contains
additional output in form of an error/warning message showing below the output cell.
This second part of the output is NOT reachable via ‘Ctrl+Shift+PgDn’,
i.e. the ‘blue frame’ focus stays with the first output cell and never 
advances to show around the error message.

2. I have to work on a position paper highlighting the “usual” struggle blind
folks go through (and they are NOT complaining about it) to get their work
in VScode done. 

3. A unified/consistent keystroke definition for the blind is needed 
but I’m sure the blind community is NOT in favor of such
a thing as for the blind folks that have painfully learned
the current jungle of keystrokes are NOT looking 
forward to learn another keystroke system…

Thanks for listening.
-Klaus.

**Comment by kds215** [comment]
This is great.  He should write  this up and send it to blind Programming.
